src/com/axis/rtspclient/RTSPClient.as

```diff
@@ -311,7 +311,9 @@ package com.axis.rtspclient {
 
         if (state === STATE_SETUP) {
           /* this is not the case when falling through, e.g. SETUP of first track */
-          if (!/RTP\/AVP\/TCP;unicast;interleaved=/.test(parsed.headers["transport"])) {
+          if (!(/^RTP\/AVP\/TCP;/.test(parsed.headers["transport"]) &&
+        	/unicast;/.test(parsed.headers["transport"]) &&
+        	/interleaved=/.test(parsed.headers["transport"]) )){
```

Comment:
I don't really understand if You want to remove or add semicolon. 
The *RTP/AVP/TCP* is always at the begining that is why it is preceded by *^* and in my opinion it can end with semicolon because there is always some parameter after it. *Interlaved* has always some parameter after = so I think it is ok now. The only problem I can see is semicolon after *unicast* - if it would be the last parameter it can have no semicolon at the end. Is that the thing that You want me to change?